packages/clangml/clangml.4.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+authors: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+bug-reports: "https://gitlab.inria.fr/memcad/clangml"
+homepage: "https://gitlab.inria.fr/memcad/clangml"
+doc: "https://gitlab.inria.fr/memcad/clangml"
+license: "BSD"
+dev-repo: "git+https://gitlab.inria.fr/memcad/clangml"
+synopsis: "OCaml bindings for Clang API"
+description: """
+clangml provides bindings to call the Clang API from OCaml.
+"""
+depends: [
+  "conf-libclang"
+  "conf-ncurses"
+  "conf-zlib"
+  "ocamlfind" {>= "1.8.0"}
```

Comment:
```suggestion
  "ocamlfind" {build & >= "1.8.0"}
```
I took a look at your code and you're using ocamlfind only in `configure.ac` however I don't really see the point of using ocamlfind if you're using dune as build system. Maybe it can be removed from the dependencies in future versions?

Reply:
Thanks for suggesting moving `ocamlfind` as build dependency. I use `ocamlfind` to check whether packages are available or not at `configure` time: even if it is not mandatory now (I could check nothing at `configure` time and fail at build time), such `configure` time checks could be useful in future version (for instance, to handle better optional dependencies). I agree that `dune` should replace most of the need of `configure`, but there are still things that I don't know how to ask `dune` to do (such as finding llvm/clang and gathering required C flags).